docs/2/installer/specific-installations/environment-monitoring-install.md

```diff
@@ -0,0 +1,64 @@
+- Protect Microshare sensors from direct sunlight, wherever possible.
+- Use Microshare solutions in environments according to their specification. Most products arefor indoor use only. 
+- There are no user-serviceable parts inside Microshare sensors. For service personnel: Always unplug external power supply, remove batteries, and turn off UPS battery before service operation. 
+- Microshare sensors may be damaged by improper storage or handling. Take care not to drop them during transport and installation.
+
+{% include image.html url="/assets/img/installation_tabs/security.png" description="Banner" %}
+
+WARNING: Do not attempt to open or disassemble any Microshare solutions. You run the risk of electric shock and voiding the limited warranty. No user-serviceable parts are inside.
+
+This information could include technical inaccuracies or typographical errors. Changes are periodically made to the information herein; these changes will be incorporated in new editions of the publication. Microshare may make improvements and/or changes in the product(s) and/or the program(s) described in this publication at any time without notice.
+
+### SE01 – Environment Monitoring
+
+The Environment Monitoring solution from Microshare is a complete sensing service for facilities managers looking to monitor the indoor air and environment quality around their buildings. 
+
+What’s in the box? In terms of deploying quickly, the Environment Monitoring solution from Microshare includes long-life battery-operated wireless sensors, which can be easily and discreetly installed around buildings, device connectivity, access to data and alerts via Microshare Business Tools dashboards or direct API access to data through the Microshare Smart Network.
+
+**Installation instructions:**
+
+| --- | --- |
+| **1.** **Activating the sensor** <br>- Our wireless sensors are pre-installed with batteries. To activate pull the shipping isolating battery tab from the sensor.<br>-	The LED on the sensor should begin to flash intermittently.<br> | {% include image.html url="/assets/img/install_environment/EnvironmentSensor.jpg" description="Environment Sensor" %} |
+| **2.** **Preparing the sensor for fixing** <br><br>- Using a suitable adhesive tape, **ensure that you place the tape on the opposite side of the QR code, covering the QR code will prevent you from using the Deploy-M app to add your device to the system.** | {% include image.html url="/assets/img/install_environment/tabs_side_adhesive.png" description="Sensor with adhesive tape" %}|
+| **3.** **Registering the sensor using the Deploy-M app [iOS & Android]** <br><br>- You should already have the Deploy-M app and login details as part of your project set-up with Microshare<br>- Login using your credentials, if you are unsure or require assistance, please contact [installation@microshare.io](mailto:installation@microshare.io) | {% include image.html url="/assets/img/leak-installation/Image12.png" description="Banner" %} |
```

Comment:
It seems like you kept the leak image here

Is it ok ? I know it's for Deploy-M

Reply:
Yes, this is correct Julien, we may update in the future, but for now we are using the same images so I thought this was more efficient....